kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -184,6 +184,9 @@ public Optional<Long> getLatestTransactionIdentifier(Engine engine, String appli
     return loadLatestTransactionVersion(engine, applicationId);
   }
 
+  /*
+   * Returns map for all active domain metadata.
+   **/
   public Map<String, DomainMetadata> getDomainMetadataMap() {
```

Comment:
`getActiveDomainMetadataMap` ?

Reply:
Updated the naming for variable and methods